1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_FizzbuzzV.2/src/index.spec.ts

```diff
@@ -8,5 +8,15 @@ describe('fizzbuzz', () => {
         it("returns '2' when given 2", () => {
         expect(fizzbuzz(2)).toBe("2")
         })
+        it("returns '100' when given 100", () => {
+            expect(fizzbuzz(100)).toBe("100")
+        })
     });
+    describe('gracefully handles out-of-bound numbers below 1 and above 100', () => {
```

Comment:
Nice. I'm planning some new modules around this, but what you've got up top here in the `describe` is what I am now calling the _Responsibility Specification_ and the  `it` is the _Concrete Example_.

Normally, the workflow will be:
- Example
- Example
- Example
- Refactor to Responsibility Specification + Parameterized Examples

But you can also work like this:
- Responsibility Specification + first example
- example
- example
- Refactor to Parameterization

Reply:
Sounds great @stemmlerjs -

Yea for this one I wanted to take the triangulation, degrees of freedrom / incrementally adding functionality to heart .